bin/citgm-all.js

```diff
@@ -56,7 +65,9 @@ var options = {
   level: app.verbose,
   npmLevel: app.npmLoglevel,
   timeoutLength: app.timeout,
-  tmpDir: app.tmpDir
+  tmpDir: app.tmpDir,
+  withTags: app.withTags,
+  excludeTags: app.excludeTags
```

Comment:
1. It seems a bit unnecessary to split `withTags` and `excludeTags` for each module, couldn't you just do it here?

```js
  withTags: app.withTags.split(','),
  excludeTags: app.excludeTags.split(',')
```

 2. Should we have some kind of logging at the start of the run case users use the wrong separator for tags, e.g. `citgm-all --withTags tag1 tag2` instead of `citgm-all --withTags tag1,tag2`. Something like:

```
Only running tests matching these 3 tags: tag1, tag2, tag3
Not running tests matching these 2 tags: tag4, tag5
```

to make it clear how many tags were found?